install.rb

```diff
@@ -108,9 +108,12 @@ def do_man(man, strip = 'man/')
     FileUtils.makedirs(om, {:mode => 0755, :verbose => true})
     FileUtils.chmod(0755, om)
     FileUtils.install(mf, omf, {:mode => 0644, :preserve => true, :verbose => true})
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
+    # Solaris does not support gzipped man pages
+    unless Facter["os"].value["family"] == "Solaris"
```

Comment:
I think `install.rb` can actually be run without requiring that `facter` be installed by passing `--no-check-prereqs`. If that's done and facter isn't installed I think this breaks.

On line 242 we already do a check for OS after confirming we aren't in 'no-prereqs' mode and assign the result to `$operatingsystem`, and then later use it to decide if we should call `do_man`. Maybe `do_man` could be extended to include an optional param of whether to compress man pages (defaulting to true), and then when calling do_man on 480 trigger that option in the negative if we're on solaris?